README.md

```diff
@@ -165,52 +166,71 @@ Confirm that the data has indeed been migrated by checking pgAdmin 4.
 
 - Export database snapshot: `pg_dump -h localhost -U USERNAME whiteclaw -f EXPORT-FILENAME.sql`
 - Wipe the existing database:
-   - Enter postgres interactively as the `postgres` user: `psql -U postgres -h localhost -W`
-   - Drop the database: `drop database whiteclaw;`
-   - Create the database: `create database whiteclaw;`
+  - Enter postgres interactively as the `postgres` user: `psql -U postgres -h localhost -W`
+  - Drop the database: `drop database whiteclaw;`
+  - Create the database: `create database whiteclaw;`
 - Restore the snapshot: `psql -h localhost -U USERNAME whiteclaw < EXPORT-FILENAME.sql`
 
+## Running Dockerized Flask App
+
+- `cd` into the root of this repo
+- Create a database dump and save it to `docker_postgres_dump.sql` using `pg_dump --create -h localhost -U <USERNAME> whiteclaw -f docker_postgres_dump.sql`
```

Comment:
is there a default data dump everyone can use instead of having to make one?

Reply:
Yeah that's a good point. Also we should give people the ability to pull dumps from prod so nobody needs to setup pg locally at all. remind me to make tickets for these items

Reply:
https://airtable.com/tbli2lWQHAqBa6ZcI/viwWvxb27XGuziw4c/recH7rMsqZQ82lEf3?blocks=hide --> ticket for this, making low priority for now